api/v1alpha1/hostedcluster_types.go

```diff
@@ -198,6 +198,11 @@ type HostedClusterSpec struct {
 	// +optional
 	SecretEncryption *SecretEncryptionSpec `json:"secretEncryption,omitempty"`
 
+	// ExposeKubeadminPassword specifies whether to expose the kubeadmin password
+	// for the guest cluster.
+	// +optional
+	ExposeKubeadminPassword bool `json:"exposeKubeadminPassword,omitempty"`
```

Comment:
If I were new to this API is not clear to me what this field is meant to do. What does it mean expose in this context? Can we elaborate a bit and specify? This is actually dictating the creation or not of an admin user right?

Reply:
Also probably rename the field accordingly